manim/utils/ipython_magic.py

```diff
@@ -161,20 +171,22 @@ def construct(self):
                 shutil.copy(local_path, tmpfile)
 
                 file_type = mimetypes.guess_type(config["output_file"])[0]
+                embed = config["media_embed"]
```

Comment:
Wouldn't it be better to add the fix for colab right here? I.e.,

```suggestion
                embed = config["media_embed"]
                if embed is None and "google.colab" in str(get_ipython()):
                    embed = True
                elif embed is None:
                    embed = False
```

I've added an additional branch to make sure that `None` is never passed to `embed` in `Image` and `Video`.

Reply:
Resolved via 1ec20177